pg_checksums.c

```diff
@@ -117,6 +118,17 @@ static const char *skip[] = {
 	NULL,
 };
 
+static void enable_progress_report(int signo,
+								   siginfo_t *siginfo,
+								   void *context) {
```

Comment:
I'd prefer if we keep using the PostgreSQL style indentation here, i.e. opening braces of functions on a new line